substrate/frame/support/procedural/src/lib.rs

```diff
@@ -1349,3 +1349,160 @@ pub fn dynamic_aggregated_params_internal(attrs: TokenStream, input: TokenStream
 		.unwrap_or_else(|r| r.into_compile_error())
 		.into()
 }
+
+/// Allows to authorize some general transactions with specific call.
```

Comment:
may be with specific function? sounds like a specific dispatchable function

Reply:
Yes, I reworded: https://github.com/paritytech/polkadot-sdk/commit/a4a2276e6bfc77b62d8b28e47cee977aa4a7a792
> /// Allows to authorize some general transactions with specific dispatchable functions
/// (dispatchable functions a.k.a. calls).